### ISSUE TITLE
[UX Planet]5 Easy Initiatives to Dramatically Improve Your App’s UX

### ISSUE THREAD

**Issue by callmemingthx** [callmemingthx/4recording.github.io#12]
### https://uxplanet.org/5-easy-initiatives-to-dramatically-improve-your-apps-ux-cf4f2fd7a8ff

if you want to create a successful app in today's ultra-competitive mobile ecosystem, you must possess a thorough understanding and fitness of User Experience.

### First, let’s wrap our heads around the term “UX.” The term has certainly become a notable buzzword, yet there is still a bit of a lacking consensus on a definition

What falls under the realm of UX? Well essentially everything that affects the customers’ end interaction and position with your product. In the case of mobile apps, this is typically usability, performance, functionality, UI, and overall design.

when it comes to your app's ux, it's important to remember that its optimization is an ongoing process. you are not expected to produce the most optimal app UX possible on the first try. however, there are certainly some simple hacks you can utilize from the get-go to ensure that your app's UX is on a direct path towards excellence.

### 1 revamp your sing-up screen
Basically, the sign-up screen is one of the key facets of your users’ first impression of your app. If this screen triggers any sort of friction between your user and your app’s UX, then it can actually spark high rates of app abandonment. 

### 2  De-scarify Your In-app Permissions
It is always a good practice to describe why your app needs certain information even if the reason might be seemingly obvious. 
It is always a good practice to describe why your app needs certain information even if the reason might be seemingly obvious. 

### 3 spice up your microinteractions
microinteractions actually have the potential to help make an app feel more human and personal. They can encourage the user to feel a connection to the brand, a connection that can ultimately improve your retention rate.
